docs/cache.txt

```diff
@@ -0,0 +1,240 @@
+.. _laravel-queues:
+
+===============
+Cache and Locks
+===============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, cache, lock, code example
+
+Configuration
+-------------
+
+To use MongoDB as a backend for `Laravel Cache and Locks <https://laravel.com/docs/{+laravel-docs-version+}/cache>`__,
+add a store configuration by using the ``mongodb`` driver in ``config/cache.php``:
+
+.. code-block:: php
+
+   'stores' => [
+       'mongodb' => [
+           'driver' => 'mongodb',
+           'connection' => 'mongodb',
+           'collection' => 'cache',
+           'lock_connection' => 'mongodb',
+           'lock_collection' => 'cache_locks',
+           'lock_lottery' => [2, 100],
+           'lock_timeout' => 86400,
+       ],
+   ],
+
+To configure the ``mongodb`` database connection, see the :ref:`laravel-fundamentals-connection` section.
+
+The following table describes a list of cache and lock options
+and their default values:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Setting
+     - Description
+
+   * - ``driver``
+     - **Required**. Specifies the lock driver to use. Must be ``mongodb``.
+
+   * - ``connection``
+     - **Required**. The database connection used to store cache items. It must be a ``mongodb`` connection.
+
+   * - ``collection``
+     - Default ``cache``. Name of the MongoDB collection to store cache items.
+
+   * - ``lock_connection``
+     - Default to the cache ``connection``. The database connection used to store locks. It must be a ``mongodb`` connection.
+
+   * - ``lock_collection``
+     - Default ``cache_locks``. Name of the MongoDB collection to store locks.
+
+   * - ``lock_lottery``
+     - Default ``[2, 100]``. Probability ``[chance, total]`` of pruning expired cache items. Set to ``[0, 0]`` to disable.
+
+   * - ``lock_timeout``
+     - Default ``86400``. Time-to-live of the locks, in seconds.
+
+
+Setup Auto-Expiration Indexes
+-----------------------------
+
+The :manual:`TTL indexes </core/index-ttl/>` integrated into MongoDB automatically
+delete documents when they expire. Their use is optional with the ``mongodb``
+driver, but recommended, because they provide better performance by delegating the
+deletion of expired documents to MongoDB instead of requiring the application to
+perform this task randomly.
```

Comment:
```suggestion
perform this task independently.
```

While the deletion is certainly random, I don't think that's relevant to the point you're trying to make here about efficiency. Alternatively, you can just say "perform this task." and leave it at that.